Client/Frontend/Onboarding/Models/OnboardingViewModel.swift

```diff
@@ -0,0 +1,10 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+struct OnboardingViewModel {
+    let cards: [OnboardingCardInfoModel]?
+    let dismissable: Bool
```

Comment:
nit, would indicate this is a Boolean with this name better I think?
```suggestion
    let isDismissable: Bool
```